README.md

```diff
@@ -1 +1,101 @@
-# morse-code
\ No newline at end of file
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Morse Decoder] <a name="about-project"></a>
+
+Morsecoder is a ruby program that takes in a morse code and decodes it to english.
+
+**[your_project__name]** is a...
+
```

Comment:
- [ ] [OPTIONAL] Kindly remove this text as it looks useless, it will make your README more professional